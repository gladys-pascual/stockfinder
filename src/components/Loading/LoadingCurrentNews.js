import PacmanLoader from "react-spinners/PacmanLoader";
import "./Loading.scss";

const LoadingCurrentNews = () => {
  return (
    <>
      <h1 className="loading-heading">Loading the latest company news...</h1>
      <div className="loading-market-news">
        <PacmanLoader color="#7bcbc4" size="60px" />
      </div>
    </>
  );
};

export default LoadingCurrentNews;
