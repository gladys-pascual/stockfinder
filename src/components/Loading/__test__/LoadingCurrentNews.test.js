import { render } from "@testing-library/react";
import LoadingCurrentNews from "../LoadingCurrentNews";

test("renders LoadingCurrentNews component as expected", () => {
  const { container } = render(<LoadingCurrentNews />);
  expect(container).toMatchSnapshot();
});
