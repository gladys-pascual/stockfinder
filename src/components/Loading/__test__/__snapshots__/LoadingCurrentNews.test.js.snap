// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LoadingCurrentNews component as expected 1`] = `
<div>
  <h1
    class="loading-heading"
  >
    Loading the latest company news...
  </h1>
  <div
    class="loading-market-news"
  >
    <span
      class="css-8pkz"
    >
      <span
        class="css-rxlor6"
      />
      <span
        class="css-13u5b26"
      />
      <span
        class="css-144dyr7"
      />
      <span
        class="css-pu3zi4"
      />
      <span
        class="css-tzx4d9"
      />
      <span
        class="css-1mjpbtg"
      />
    </span>
  </div>
</div>
`;
